src/main/kotlin/no/nav/familie/ef/iverksett/domene/AndelTilkjentYtelse.kt

```diff
@@ -1,18 +1,14 @@
 package no.nav.familie.ef.iverksett.domene
 
-import no.nav.familie.kontrakter.ef.felles.StønadType
-import java.util.*
+import java.util.UUID
 
 data class AndelTilkjentYtelse(val periodebeløp: Periodebeløp,
-                               val personIdent: String,
                                val periodeId: Long? = null,
                                val forrigePeriodeId: Long? = null,
-                               val stønadsType: StønadType? = null,
```

Comment:
Er ikke nødvendigvis uenig, men vil bare sjekke at vi er helt sikre på at der er en god idé:

- Nå kan vi ikke lenger ha én tilkjent ytelse som dekker flere stønadstyper. Det betyr f.eks at tilkjent overgangsstønad og  tilkjente skolepenger er tilknyttet to tilkjente ytelser, en for hver stønadstype
- Det er ikke lenger mulig å knytte en andel til en annen person enn søker

Reply:
- Sant, men kommer vi noen gang til å ha knytte forskjellige stønadstyper til samme tilkjente ytelse? I Sak er jo dette delt på fagsaknivå som igjen har hver sin behandling og tilkjentYtelse?
- Godt poeng, men skal det knyttes noen andeler til andre personer enn søkeren for vår del? Barnetilsyn krever jo at man har utgifter for enkeltbarn, men disse skal uansett knyttes til søkeren i utbetalingsøyemed. Dog er det kanskje interessant å vite hvilke av barna du mottar utbetalingen for, men vet ikke om personIdent er riktig sted å mappe barna?

@blommish @ma10s: Har dere noen tanker rundt dette?

Reply:
1. Trenger vi det? Har ikke hørt noe om att vi har dette behovet
2. Er det ikke kun søkeren som skal få ytelsen?

Reply:
Jeg er enig, jeg tror ikke vi har disse behovene, men ta gjerne en sjekk med funksjonelle.